packages/rust-analyzer/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rust-analyzer/rust-analyzer
+TERMUX_PKG_DESCRIPTION="A Rust compiler front-end for IDEs"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE_FILE="LICENSE-MIT"
```

Comment:
It is . Without it build fails at `termux_step_install_licence` . Note that step by default looks for `LICENCE` and a few other commonly found licence files in repo . But in case of rust-analyzer they have it under dual lincence that's why they have two separate apace and mit licence files

Reply:
You can see the following example for dual license...
https://github.com/termux/termux-packages/blob/0985b1426413d8a04423e46ba3bddde87976ee2a/packages/bmon/build.sh#L4

Reply:
Thanks. I've Updated it to have both MIT and APACHE licence :)